forms/jqm/section.js

```diff
@@ -57,6 +57,14 @@ define(function (require) {
       delete this.$el;
       delete this.el;
       return result;
+    },
+    onAttached: function() {
+      this.model.get('elements').models.forEach(function (el, i) {
```

Comment:
I think ESLint is complaining about the unused `i` parameter, which is breaking Travis CI builds.